src/lib.js

```diff
@@ -253,11 +257,10 @@ export default class Migrator {
           migrationName = migrationToImport.slice(timestampSeparatorIndex + 1, migrationToImport.lastIndexOf('.'));
 
         this.log(`Adding migration ${filePath} into database from file system. State is ` + `DOWN`.red);
-        const createdMigration = MigrationModel.create({
+        return MigrationModel.create({
```

Comment:
@npahucki that seems to hint to me that you're not using babel to transpile the code?